test-images/ubuntu-bionic/Dockerfile

```diff
@@ -56,7 +56,7 @@ USER root
 RUN apt-get install -y  build-essential \
                     libcurl4-openssl-dev \
                     libssl-dev \
-                    zlib1g-dev zlibc \
+                    zlib1g-dev  \
```

Comment:
Since zlibc is deprecated, I removed from the installation list for ubuntu distros. Caused error for ubuntu jammy, since this package does not exist in ubuntu jammy repo.